python/adbc_driver_flightsql/tests/test_errors.py

```diff
@@ -184,9 +184,3 @@ def test_query_error_getflightinfo(test_dbapi):
             ),
         ):
             cur.adbc_execute_partitions("error_get_flight_info")
-
-
-def test_stateless_prepared_statement(test_dbapi) -> None:
-    with test_dbapi.cursor() as cur:
-        cur.adbc_prepare("stateless_prepared_statement")
-        cur.execute("stateless_prepared_statement", parameters=[(1,)])
```

Comment:
I know this is a flaky test, but can we mark with skip instead of deleting it?

Reply:
I've tried to add `@pytest.mark.skip` on top of the failing test function but I got `Forbidden skipped test`. Is there any global config that forbid skipping tests?

Reply:
Ah that's right, CI runs with a plugin that prevents us from accidentally skipping tests.

@zeroshade this is still outstanding.

I would just ignore the test for now.

Reply:
@lidavidm what's still outstanding?

Reply:
The flaky test (the EOF issue)